Ark.Tools.AspNetCore/ProblemDetails/ArkProblemDetailsOptionsSetup.cs

```diff
@@ -110,7 +112,7 @@ private Microsoft.AspNetCore.Mvc.ProblemDetails _toProblemDetails(BusinessRuleVi
             });
 
             var js = (arg.BusinessRuleViolation as object).SerializeToByte(ArkSerializerOptions.JsonOptions);                
-            var ret = (Microsoft.AspNetCore.Mvc.ProblemDetails)JsonSerializer.Deserialize(js, pdt, ArkSerializerOptions.JsonOptions);
+            var ret = (Microsoft.AspNetCore.Mvc.ProblemDetails)JsonSerializer.Deserialize(js, pdt, ArkSerializerOptions.JsonOptions)!;
```

Comment:
Why the "!" after?

Reply:
since arg.BusinessRuleViolation is not null here, the Deserialization is safe to be asserted as not null. 
Compiler cannot know it, but it's sure to be since this is just a 'trick' to transform a BRV in a MVC.ProblemDetails